manifests/s/ShiningLight/OpenSSL/3.1.0/ShiningLight.OpenSSL.installer.yaml

```diff
@@ -0,0 +1,53 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL
+PackageVersion: 3.1.0
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+Scope: machine
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x86

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

Reply:
Since only the `x86` packages need the dependency, I'm suggesting it be added only to those.